Code/wwlib/cpudetect.cpp

```diff
@@ -824,7 +824,7 @@ void CPUDetectClass::Init_CPUID_Instruction()
    // because CodeWarrior seems to have problems with
    // the command (huh?)
 
-#ifdef _M_IX86
+#if defined(_MC_VER) && defined(_M_IX86)
```

Comment:
Could probably just dump the ASM and set it to true here, I don't think Renegade min requirements were a CPU without it.

Reply:
Most of the in-line assembly is gone.